src/root/components/Checklist/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import InputLabel from '#components/InputLabel';
+import InputError from '#components/InputError';
+import List from '#components/List';
+import Checkbox from '#components/Checkbox';
+
+import styles from './styles.module.scss';
+
+type OptionKey = string | number;
+
+export interface Props<V, O, N> {
+  className?: string;
+  name: N;
+  options: O[];
+  onChange: (value: V[] | undefined, name: N) => void;
+  value: V[] | undefined;
```

Comment:
```suggestion
  value: V[] | null | undefined;
```
Let's also support null as value even if we don't set "null" value